include/bluetooth/iso.h

```diff
@@ -245,6 +342,41 @@ int bt_iso_chan_disconnect(struct bt_iso_chan *chan);
  */
 int bt_iso_chan_send(struct bt_iso_chan *chan, struct net_buf *buf);
 
+/** @brief Creates a BIG as a broadcaster
+ *
+ *  @param padv    Pointer to the periodic advertising object the BIGInfo shall
+ *                 be sent on.
+ *  @param param   The parameters used to create and enable the BIG. The QOS
+ *                 parameters are determined by the QOS field of the first BIS
+ *                 in the BIS list of this parameter.
+ *
+ *  @return 0 in case of success or negative value in case of error.
+ */
+int bt_iso_create_big(struct bt_le_ext_adv *padv,
+		      struct bt_iso_create_big_param_t *param);
+
```

Comment:
I would attemp to map the parameters to bt_iso_channel as well, so we can use function like bt_iso_disconnect to terminate, etc, also perhaps we should name this bt_iso_broadcast(channel, padv) then the channel would contain qos with the related field to create a the big. If this all can be arranged it would also be possible to use bt_iso_chan_send to transmit over broadcast.

Reply:
`bt_iso_chan_send` is already used to send over broadcast. 

Regarding `bt_iso_terminate` that should also be possible to implement. The reason why I haven't done that, is that I don't see a use case where you'd want to disconnect a single BIS in a BIG (actually I don't understand the usecase for disconnecting a single CIS in a CIG). Do you have an example of where you'd might want to do that? 

I'm not sure what other parameters you are referring to here.